main.js

```diff
@@ -5,6 +5,16 @@ const BrowserWindow = electron.BrowserWindow;
 const path = require('path');
 const fs = require('fs');
 
+if (process.env.NODE_ENV != 'production') {
```

Comment:
What is this?

Reply:
https://github.com/rotkehlchenio/rotkehlchen/pull/136#discussion_r213510227